core/pkg/launch/job_builder.go

```diff
@@ -702,12 +702,25 @@ func (j *JobBuilder) HandleUseArtifactRecord(record *service.Record) {
 func (j *JobBuilder) MakeJobMetadata(output *data_types.TypeRepresentation) (string, error) {
 	metadata := make(map[string]interface{})
 	input_types := make(map[string]interface{})
+	input_schemas := make(map[string]interface{})
```

Comment:
replace `make(map[string]interface{})` -> `make(map[string]any)`

@timoffex we probably want to lint for this in our codebase